index.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Please remove the code to disable the linter and fix linter errors manually, It will help you to write clean code.